app/jobs/gallery_validation_job.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+##
+# Validate that a gallery has only properly displaying images
+# Should an issue be discovered, a slack channel will be notified by sending an email.
+class GalleryValidationJob < ApplicationJob
+  queue_as :default
+
+  def perform(gallery_id)
+    @gallery = Gallery.find(gallery_id)
```

Comment:
`fail` here if `Rails.application.config.x.gallery_validation_mail_to` is not set.